test/ejabberd_tests/tests/privacy_SUITE.erl

```diff
@@ -470,12 +471,15 @@ allow_subscription_to_from_message(Config) ->
 
         %% Alice subscribes to Bob
         escalus_client:send(Alice,
-                            escalus_stanza:presence_direct(bob, <<"subscribe">>)),
+            escalus_stanza:presence_direct(escalus_utils:get_short_jid(Bob),
```

Comment:
Are you sure @RoXeon, you replied to the correct PR? :) Despite that, it sounds like a plan :)